beacon_node/lighthouse_network/src/discovery/enr.rs

```diff
@@ -221,6 +238,16 @@ pub fn build_enr<E: EthSpec>(
 
     builder.add_value(SYNC_COMMITTEE_BITFIELD_ENR_KEY, &bitfield.as_ssz_bytes());
 
+    // only set `csc` if PeerDAS fork epoch has been scheduled
+    if spec.eip7594_fork_epoch != Some(spec.far_future_epoch) {
```

Comment:
Same comment here

Reply:
Fixed in https://github.com/sigp/lighthouse/pull/6216/commits/94f1d3dd93622b358e52b1ee79febbed67292c84